drivers/flash/flash_qspi_renesas_ra.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2024 Renesas Electronics Corporation
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_DRIVERS_FLASH_QSPI_RENESAS_RA_H_
+#define ZEPHYR_DRIVERS_FLASH_QSPI_RENESAS_RA_H_
+
+#include <zephyr/drivers/flash.h>
+#include <api/r_spi_flash_api.h>
+#include <zephyr/drivers/flash/ra_flash_api_extensions.h>
+
+#define QSPI_CMD_RESET_EN       (0x66) /* Reset Enable */
+#define QSPI_CMD_RESET_MEM      (0x99) /* Reset Memory */
+#define Ex_SPI_CMD_READ_ID      (0x9F) /* Extended Spi Read JEDEC ID */
+#define QSPI_CMD_READ_ID        (0xAF) /* Qpi Read JEDEC ID */
+#define QSPI_CMD_READ_SFDP      (0x5A) /* Read SFDP */
```

Comment:
Most of those defines are actually provided here https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/spi_nor.h, I know the naming is SPI_NOR, but adding additional defines only brings new names for functionally the same things.
You could just use the SPI_NOR defines and define these that are not provided by the header.

Reply:
We changed to use macros in https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/flash/spi_nor.h